### ISSUE TITLE
Is the project dead?

### ISSUE THREAD

**Issue by w0rldart** [acrobox/abx#6]
Just stumbled across this project on https://www.indiehackers.com/post/how-are-you-hosting-your-sites-6528c79748, but the website is empty and this seems abandoned.

**Comment by pnelson** [comment]
Yes, dead and abandoned - earned my lessons learned. Maybe one day I'll spin something off of it.

**Comment by w0rldart** [comment]
Thanks for the update @pnelson! I'd be very keen to learn about your lessons learned, if you'd be willing to share. One can learn for other people's experience, to avoid repeating certain mistakes, or just fly through the learning process.

**Comment by pnelson** [comment]
Oh, way too many. A non-exhaustive list from top of mind would include not getting the MVP out early and often, the most basic of advice I had probably heard hundreds if not thousands of times but needed to learn through direct experience.

There were several factors that lead to that but I think a big one was that I was chasing long term MRR before I made my first dollar.

I wanted to make my first dollar before I quit my job but that didn't work either. I was burnt out. I couldn't do another day. The moment I took that leap ignited the flame to take action and with it came a sense of purpose and inflated confidence. I was free fallin', baby.

The high of erupting from my shell, sprinkled with a little pandemic-induced mass psychosis, and I was on another level. I took the reigns of my life. Everything was possible. I took on a little too much at once, both underestimating the difficulty of my life changes and overestimating my capabilities.

This project was intended to be open source and used to build a following, a name for myself, and become the tool for which I'd rapidly build, deploy, and iterate my paid products on.

The project was usable, tested, and documented in that that state about 8 months into my journey. A sizable chunk of that time was spent trying to both finalize divorce and immigrate during the early stages of the pandemic, documenting the journey, and processing the tsunami of data pouring into my psyche.

I worked so damn hard on this project and started to see a product vision. I see product ideas everywhere I look but this was my first time without income - I was hungry but naive. I think I let my fear rationalize this pivot from my original vision. I took it a step further by ignoring the many clear signs that a one time fee to provision a machine would have been more suitable profit model for this service. This model would also have reduced the implementation overhead to get to market. I needed to have this mindset to begin with but my time was more or less up.

By this point I was sufficiently distracted with a 6 month intensive Mandarin course, a separate life goal of mine that I decided to pursue simultaneously (RE: inflated confidence). Following that came navigating visa issues and eventually facing that I did not achieve what I set out to do and reflecting on areas where I went wrong.

There are many meta lessons in the above.

I will have the backing weight of my earned wisdom the next time I execute on a vision.

**Comment by w0rldart** [comment]
@pnelson sounds like many beautiful things (though hard, and funnily enough... I can relate to every single thing) have happened for you, and not to you. That's awesome, because now you poses an experience, and internal alignment to fuel you personally and the future projects that you'll bring to life.

So, I can only say... congratulations, and thank you very much for sharing this with me.

**Comment by w0rldart** [comment]
Keep me posted about your future projects, I'll follow them closely
